toolkit/javascripts/user-research-consent-banner.js

```diff
@@ -0,0 +1,56 @@
+;(function (global) {
+  "use strict";
+
+  var UserResearchBanner = function( _userResearchBanner ) {
+    this.$userResearchBanner = _userResearchBanner;
+    this.$userResearchBannerCloseBtn = this.$userResearchBanner.find('.user-research-banner-close-btn');
+    this.userResearchCookie = {
+      name: 'seen_user_research_message',
+      value: true
+    };
+    this.showBannerIfMessageNotSeen();
+    this.initaliseListener();
+  };
+  
+  UserResearchBanner.init = function () {
+    
+  };
+
+  UserResearchBanner.prototype.initaliseListener = function () {
+    var _this = this;
+    this.$userResearchBannerCloseBtn.on('click', function () {
+      _this.setCookie();
+      _this.hideBanner();
+    });
+  };
+
+  UserResearchBanner.prototype.showBannerIfMessageNotSeen = function () {
+    if (!this.getCookie()) {
+      this.showBanner();
+    }
+  };
+
+  UserResearchBanner.prototype.showBanner = function () {
+    this.$userResearchBanner.show();
+  };
+
+  UserResearchBanner.prototype.hideBanner = function () {
+    this.$userResearchBanner.hide();
+  };
+
+  UserResearchBanner.prototype.getCookie = function () {
```

Comment:
Will this cause the banner to show if the user isn't logged in?

Reply:
The template should not be inserted if the user is anonymous. Not sure if there is some JS helper that allows us to see if the user is logged in or not